### ISSUE TITLE
Why pretrained model is required for SAM-GRad CAM?

### ISSUE THREAD

**Issue by ChidanandKumarKS** [Intelligent-Computing-Lab-Yale/Visual-Explanations-from-Spiking-Neural-Networks-using-Interspike-Intervals#1]
@youngryan1993 
As i understood from the paper, Surrogate gradient learning found to be performing better than ANN-SNN conversion technique. Then why training requires pretrained model? Kindly clarify
